### ISSUE TITLE
pip 1.5.0 package install breaks

### ISSUE THREAD

**Issue by rday** [LudovicRousseau/PyKCS11#32]
#### Your system information

* PyKCS11 version: 1.5.0
* Python version: 3.6.2
* Os: ALinux

#### Please describe your issue in as much detail as possible:

When using pip to upgrade to the PyKCS11 1.5.0 package on Linux, I noticed a few errors around the LowLevel library. When examining the previous PyPI package(1.4.4), it looks like the LowLevel module and the pykcs11_wrap.cpp code are included in the package. This is not the case with the 1.5.0 package.

The pip installation appears to succeed, but I am unable to import the PyKCS11 library.

Are there new install steps, or could this be a bad pip package?

#### Steps for reproducing this issue:

```
(py-env) [ec2-user@ip-172-31-31-126 la]$ pip --no-cache-dir -v install PyKCS11
Collecting PyKCS11
  1 location(s) to search for versions of PyKCS11:
  * https://pypi.python.org/simple/pykcs11/
  Getting page https://pypi.python.org/simple/pykcs11/
  Starting new HTTPS connection (1): pypi.python.org
  https://pypi.python.org:443 "GET /simple/pykcs11/ HTTP/1.1" 200 1439
  Analyzing links from page https://pypi.python.org/simple/pykcs11/
    Skipping link https://pypi.python.org/packages/e9/f4/e5b42ab9f9049cbb8849aa6c9add3adff8889798c87cf9edb8ff4d4e6e45/PyKCS11-1.1.1.win32-py2.4.exe#md5=4778e61316b9e209684ac057f70d90d6 (from https://pypi.python.org/simple/pykcs11/); unsupported archive format: .exe
    Found link https://pypi.python.org/packages/28/1e/091d5fcba391eaac36edb26826be92c72b4ae5b3e25f6379de5401f3b335/PyKCS11-1.1.1.zip#md5=3481059ab83f49d2caf30201c995ae1f (from https://pypi.python.org/simple/pykcs11/), version: 1.1.1
    Found link https://pypi.python.org/packages/ab/11/cc2a85f528e3bf5e39a1da889070bf797fcf010eb2de9b581cdc49488a23/PyKCS11-1.2.0.tar.gz#md5=cd06a0e9e0f80f176066befd08505b1a (from https://pypi.python.org/simple/pykcs11/), version: 1.2.0
    Skipping link https://pypi.python.org/packages/7f/0e/64dded7d29a84b34bd42cb83ce7a4aeb1e5cb02a23742d676c1b19e54241/PyKCS11-1.2.0.win32-py2.4.exe#md5=a2ac5c274f798838cd6c6ed3bdb92cec (from https://pypi.python.org/simple/pykcs11/); unsupported archive format: .exe
    Found link https://pypi.python.org/packages/25/1c/8dfba4d52784c282df072887c5eaf5ee7077e1acdbe8cbd7692e32a33e86/PyKCS11-1.2.1.tar.gz#md5=f1be0bcef765a36e4362342c5df49eef (from https://pypi.python.org/simple/pykcs11/), version: 1.2.1
    Skipping link https://pypi.python.org/packages/e8/ed/500105a37d79846d3077b118dc691707d85866a56ebd76c03fe2aa8605cd/PyKCS11-1.2.1.win32-py2.4.exe#md5=9e176dc678ed76e00f26a34e4ebfc034 (from https://pypi.python.org/simple/pykcs11/); unsupported archive format: .exe
    Skipping link https://pypi.python.org/packages/57/cb/37896a4607c81d7351a2d40be720e68088c3e13b2a4ffbe76ea276398d86/PyKCS11-1.2.1.win32-py2.5.exe#md5=9b9c2aba1d16ab993ee5b9e60dd00942 (from https://pypi.python.org/simple/pykcs11/); unsupported archive format: .exe
    Found link https://pypi.python.org/packages/7c/d7/2ccbf18ef020545933b096a035973146138f8a830272a0a555ae258ad8fb/PyKCS11-1.2.4.tar.gz#md5=39f85ed9e30719819ede4db5345996d0 (from https://pypi.python.org/simple/pykcs11/), version: 1.2.4
    Found link https://pypi.python.org/packages/e5/a4/02b2836103348d38abb793ed2461386dd2c7e652a03a6e295ce569e0ca19/PyKCS11-1.3.0.tar.gz#md5=db1dffd14ed7f5860a2403b0097b6933 (from https://pypi.python.org/simple/pykcs11/), version: 1.3.0
    Found link https://pypi.python.org/packages/70/97/ee9cefcd6052ae436be2b91efa59610421a1e917506e01aaf29cb3727ad4/PyKCS11-1.3.1.tar.gz#md5=e7690765736025888ecbce70229d2064 (from https://pypi.python.org/simple/pykcs11/), version: 1.3.1
    Found link https://pypi.python.org/packages/c6/db/5e3ad4c6f24795844f4550cd8bdacf54f4061840613de81f0267a05692dc/PyKCS11-1.3.2.tar.gz#md5=7a90264871449e522558fddc1bf9f94f (from https://pypi.python.org/simple/pykcs11/), version: 1.3.2
    Found link https://pypi.python.org/packages/5f/e2/61fd2636d6e4629600ce677139c5d4582a6d3f774bbc7e7a8ca94a7ec50a/PyKCS11-1.3.3.tar.gz#md5=c8d74043a97aef4fdfe1cbd527069c01 (from https://pypi.python.org/simple/pykcs11/), version: 1.3.3
    Found link https://pypi.python.org/packages/c4/91/55ca0aa59d56c59b895a0cc48ee53495f609057e494571d331c34868fd38/PyKCS11-1.4.0.tar.gz#md5=2ef21e296b9b4e330f7adce8d89cc95b (from https://pypi.python.org/simple/pykcs11/), version: 1.4.0
    Found link https://pypi.python.org/packages/6f/ab/23c358a6e6af5e396fb8817e2da05a489891d3d20a103e34f8a56180a395/PyKCS11-1.4.1.tar.gz#md5=b0f2c1dda012a64a2376aca2bcd4afb3 (from https://pypi.python.org/simple/pykcs11/), version: 1.4.1
    Found link https://pypi.python.org/packages/f6/94/2e09d543049afa4b14f9f2820a2a4f297ff4e33ff64b1092ed7961f93a4c/PyKCS11-1.4.2.tar.gz#md5=cd39e136bb0b2341f07e029a6b0b9a99 (from https://pypi.python.org/simple/pykcs11/), version: 1.4.2
    Found link https://pypi.python.org/packages/17/0e/0cf98fe7e2509b81f484d724f875934627fba7b00ac1af96b81bc6b193c2/PyKCS11-1.4.3.tar.gz#md5=d3141d75154c89afbf230280f28ef24f (from https://pypi.python.org/simple/pykcs11/), version: 1.4.3
    Found link https://pypi.python.org/packages/6e/65/2421a94aa553cda9d3b6580f9d49e6d58625545cd650f09dbdd68e19f394/PyKCS11-1.4.4.tar.gz#md5=cdf42fc3c5a28dca05ffb2ffd1ea085b (from https://pypi.python.org/simple/pykcs11/), version: 1.4.4
    Found link https://pypi.python.org/packages/7c/9b/22725f013616b9613e015d4206aca006204b9f448100f803c15c7e99ab15/PyKCS11-1.5.0.tar.gz#md5=73c914613a3be3f04df6737cbf02b507 (from https://pypi.python.org/simple/pykcs11/), version: 1.5.0
  Using version 1.5.0 (newest of versions: 1.1.1, 1.2.0, 1.2.1, 1.2.4, 1.3.0, 1.3.1, 1.3.2, 1.3.3, 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.5.0)
  https://pypi.python.org:443 "GET /packages/7c/9b/22725f013616b9613e015d4206aca006204b9f448100f803c15c7e99ab15/PyKCS11-1.5.0.tar.gz HTTP/1.1" 200 63667
  Downloading PyKCS11-1.5.0.tar.gz (63kB)
  Downloading from URL https://pypi.python.org/packages/7c/9b/22725f013616b9613e015d4206aca006204b9f448100f803c15c7e99ab15/PyKCS11-1.5.0.tar.gz#md5=73c914613a3be3f04df6737cbf02b507 (from https://pypi.python.org/simple/pykcs11/)
    100% |████████████████████████████████| 71kB 15.9MB/s 
  Running setup.py (path:/tmp/pip-build-qcwilyw4/PyKCS11/setup.py) egg_info for package PyKCS11
    Running command python setup.py egg_info
    running egg_info
    creating pip-egg-info/PyKCS11.egg-info
    writing pip-egg-info/PyKCS11.egg-info/PKG-INFO
    writing dependency_links to pip-egg-info/PyKCS11.egg-info/dependency_links.txt
    writing top-level names to pip-egg-info/PyKCS11.egg-info/top_level.txt
    writing manifest file 'pip-egg-info/PyKCS11.egg-info/SOURCES.txt'
    warning: manifest_maker: standard file '-c' not found

    file PyKCS11/LowLevel.py (for module PyKCS11.LowLevel) not found
    reading manifest file 'pip-egg-info/PyKCS11.egg-info/SOURCES.txt'
    writing manifest file 'pip-egg-info/PyKCS11.egg-info/SOURCES.txt'
  Source in /tmp/pip-build-qcwilyw4/PyKCS11 has version 1.5.0, which satisfies requirement PyKCS11 from https://pypi.python.org/packages/7c/9b/22725f013616b9613e015d4206aca006204b9f448100f803c15c7e99ab15/PyKCS11-1.5.0.tar.gz#md5=73c914613a3be3f04df6737cbf02b507
Installing collected packages: PyKCS11
  Running setup.py install for PyKCS11 ...     Running command /home/ec2-user/py-env/bin/python3.6 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-qcwilyw4/PyKCS11/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-ni7_l6ig-record/install-record.txt --single-version-externally-managed --compile --install-headers /home/ec2-user/py-env/include/site/python3.6/PyKCS11
    running install
    running build
    running build_py
    file PyKCS11/LowLevel.py (for module PyKCS11.LowLevel) not found
    creating build
    creating build/lib.linux-x86_64-3.6
    creating build/lib.linux-x86_64-3.6/PyKCS11
    copying PyKCS11/__init__.py -> build/lib.linux-x86_64-3.6/PyKCS11
    file PyKCS11/LowLevel.py (for module PyKCS11.LowLevel) not found
    running build_ext
    building 'PyKCS11._LowLevel' extension
    swigging src/pykcs11.i to src/pykcs11_wrap.cpp
    swig -python -c++ -o src/pykcs11_wrap.cpp src/pykcs11.i
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/src
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -Isrc -I/usr/local/include -I/home/ec2-user/py-env/include -I/home/ec2-user/py362/include/python3.6m -c src/ck_attribute_smart.cpp -o build/temp.linux-x86_64-3.6/src/ck_attribute_smart.o
    cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++ [enabled by default]
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -Isrc -I/usr/local/include -I/home/ec2-user/py-env/include -I/home/ec2-user/py362/include/python3.6m -c src/pkcs11lib.cpp -o build/temp.linux-x86_64-3.6/src/pkcs11lib.o
    cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++ [enabled by default]
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -Isrc -I/usr/local/include -I/home/ec2-user/py-env/include -I/home/ec2-user/py362/include/python3.6m -c src/pykcs11string.cpp -o build/temp.linux-x86_64-3.6/src/pykcs11string.o
    cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++ [enabled by default]
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -Isrc -I/usr/local/include -I/home/ec2-user/py-env/include -I/home/ec2-user/py362/include/python3.6m -c src/utility.cpp -o build/temp.linux-x86_64-3.6/src/utility.o
    cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++ [enabled by default]
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -Isrc -I/usr/local/include -I/home/ec2-user/py-env/include -I/home/ec2-user/py362/include/python3.6m -c src/pykcs11_wrap.cpp -o build/temp.linux-x86_64-3.6/src/pykcs11_wrap.o
    cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++ [enabled by default]
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -Isrc -I/usr/local/include -I/home/ec2-user/py-env/include -I/home/ec2-user/py362/include/python3.6m -c src/pykcs11.cpp -o build/temp.linux-x86_64-3.6/src/pykcs11.o
    cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++ [enabled by default]
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -Isrc -I/usr/local/include -I/home/ec2-user/py-env/include -I/home/ec2-user/py362/include/python3.6m -c src/dyn_unix.c -o build/temp.linux-x86_64-3.6/src/dyn_unix.o
    g++ -pthread -shared build/temp.linux-x86_64-3.6/src/ck_attribute_smart.o build/temp.linux-x86_64-3.6/src/pkcs11lib.o build/temp.linux-x86_64-3.6/src/pykcs11string.o build/temp.linux-x86_64-3.6/src/utility.o build/temp.linux-x86_64-3.6/src/pykcs11_wrap.o build/temp.linux-x86_64-3.6/src/pykcs11.o build/temp.linux-x86_64-3.6/src/dyn_unix.o -L/usr/local/lib -o build/lib.linux-x86_64-3.6/PyKCS11/_LowLevel.cpython-36m-x86_64-linux-gnu.so
    running install_lib
    creating /home/ec2-user/py-env/lib/python3.6/site-packages/PyKCS11
    copying build/lib.linux-x86_64-3.6/PyKCS11/_LowLevel.cpython-36m-x86_64-linux-gnu.so -> /home/ec2-user/py-env/lib/python3.6/site-packages/PyKCS11
    copying build/lib.linux-x86_64-3.6/PyKCS11/__init__.py -> /home/ec2-user/py-env/lib/python3.6/site-packages/PyKCS11
    byte-compiling /home/ec2-user/py-env/lib/python3.6/site-packages/PyKCS11/__init__.py to __init__.cpython-36.pyc
    running install_egg_info
    running egg_info
    creating PyKCS11.egg-info
    writing PyKCS11.egg-info/PKG-INFO
    writing dependency_links to PyKCS11.egg-info/dependency_links.txt
    writing top-level names to PyKCS11.egg-info/top_level.txt
    writing manifest file 'PyKCS11.egg-info/SOURCES.txt'
    warning: manifest_maker: standard file '-c' not found

    file PyKCS11/LowLevel.py (for module PyKCS11.LowLevel) not found
    reading manifest file 'PyKCS11.egg-info/SOURCES.txt'
    writing manifest file 'PyKCS11.egg-info/SOURCES.txt'
    Copying PyKCS11.egg-info to /home/ec2-user/py-env/lib/python3.6/site-packages/PyKCS11-1.5.0-py3.6.egg-info
    running install_scripts
    file PyKCS11/LowLevel.py (for module PyKCS11.LowLevel) not found
    writing list of installed files to '/tmp/pip-ni7_l6ig-record/install-record.txt'
done
  Removing source in /tmp/pip-build-qcwilyw4/PyKCS11
Successfully installed PyKCS11-1.5.0
Cleaning up...
(py-env) [ec2-user@ip-172-31-31-126 la]$ python
Python 3.6.2 (default, Sep  1 2017, 16:56:32) 
[GCC 4.8.3 20140911 (Red Hat 4.8.3-9)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import PyKCS11
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ec2-user/py-env/lib/python3.6/site-packages/PyKCS11/__init__.py", line 20, in <module>
    import PyKCS11.LowLevel
ModuleNotFoundError: No module named 'PyKCS11.LowLevel'
>>> quit()
```

**Comment by romaia** [comment]
I am too, affected by this problem. LowLevel.py is missing. Had to downgrade to 1.4.4 to get pykcs11 working again.

**Comment by caduellery** [comment]
I am too, affected by this problem. Solved like @romaia with:

pip install pykcs11=1.4.4

**Comment by LudovicRousseau** [comment]
I can also reproduce the problem.
I will work on it but patches are always welcome.

If I install it using ``./setup.py install`` I have:
```
$ find /usr/local/lib/python2.7/site-packages/PyKCS11
/usr/local/lib/python2.7/site-packages/PyKCS11
/usr/local/lib/python2.7/site-packages/PyKCS11/_LowLevel.so
/usr/local/lib/python2.7/site-packages/PyKCS11/__init__.py
/usr/local/lib/python2.7/site-packages/PyKCS11/LowLevel.py
/usr/local/lib/python2.7/site-packages/PyKCS11/__init__.pyc
/usr/local/lib/python2.7/site-packages/PyKCS11/LowLevel.pyc
```

With ``pip --no-cache-dir -v install PyKCS11`` I get:
```
$ find /usr/local/lib/python2.7/site-packages/PyKCS11
/usr/local/lib/python2.7/site-packages/PyKCS11
/usr/local/lib/python2.7/site-packages/PyKCS11/_LowLevel.so
/usr/local/lib/python2.7/site-packages/PyKCS11/__init__.py
/usr/local/lib/python2.7/site-packages/PyKCS11/__init__.pyc
```

Note that the files ``LowLevel.py`` and ``LowLevel.pyc`` are missing.

**Comment by LudovicRousseau** [comment]
Fixed in release 1.5.1
https://github.com/LudovicRousseau/PyKCS11/releases/tag/1.5.1
